src/core/shaderlib/project/project.glsl.js

```diff
@@ -70,7 +72,12 @@ vec3 project_normal(vec3 vector) {
     project_uCoordinateSystem == COORDINATE_SYSTEM_LNGLAT_OFFSETS) {
     return normalize(vector * project_uPixelsPerDegree);
   }
-  return normalize(vector * project_uPixelsPerUnit);
+  return normalize(vector * project_uPixelsPerMeter);
+}
+
+vec4 project_offset(vec4 offset) {
```

Comment:
Is this an internal method? Should it have an underscore at the end like `project_mercator_`?

Maybe call these
```
project_mercator_position_()
project_offset_position_()
```